setup.cfg

```diff
@@ -28,11 +28,12 @@ install_requires =
     defusedxml
     distro
     jinja2
+    pip>=21.1
```

Comment:
During deployment on older OSes invirtualenv needs to be able to run with the version of pip the OS uses so we can't add a requirement for pip without breaking stuff.